starter-code/contact-app/src/app/contact-list/contact-list.component.html

```diff
@@ -15,4 +15,13 @@
   <input type="text" [(ngModel)]="newContact.image">
 </div>
 
-<button (click)="addContact">Submit New Contact</button>
+<p *ngFor="let contact of contacts">
+  {{contact.name}}
```

Comment:
Aquí tanto como en su formulario pudieron poner labels para darle contexto a que es la información y que estás escribiendo en cada input, ejemplo: `Nombre: {{ contact.name }}` y `
<label for="contact-email">Email</label>`